kythe/proto/explore.proto

```diff
@@ -159,44 +157,37 @@ message TypeHierarchyReply {
 
 // Call graph
 // These APIs are for capturing the graphs induced by the _semantic_ relations between
-// functions (that is, the functions that call, or are called by, this function).
+// calling/callable nodes (for example, the functions that call, or are called by,
+// this function node).
 //
 // If you’re looking for a one-ply relationship between functions and the
 // _locations_ in the code at which they are called, Kythe XRefService's
 // CrossReferences API should do what you want.
 
-// Requests the functions that call the specified function.
+// Requests the incoming callgraphs for each of the specified nodes.
```

Comment:
Do we require at least 1 requested ticket?

What about a maximum allowed?

(Same for CalleesRequest below)

Reply:
We do require at least 1 input ticket.  If you think that users might reasonably expect otherwise I could add a comment to that effect.

We've got plumbing to impose a limit on the max number of tickets, but it's not yet enabled.  In any event I'm not sure we want to document that limit here, since it's configurable.